### DISCUSSION TITLE
Can I ONLY show the pages that have been promoted/posted as News inside the "PnP - Search Results"

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#1511] [Q&A]
I added the PnP search SPFx web parts. Inside the "PnP - Search Results" web part, i defined the following to show SharePoint Pages, as per the below 2 images:-

https://ibb.co/PhqRF5H

https://ibb.co/BGgfcwn

The above is working well where it is retrieving all the SharePoint pages as per the "Refinement filters" (**_FileType:equals("aspx")_**).. but is there a way to do extra filter/refiner to ONLY show the Pages that have been promoted/posted as News, and not to show all the Pages (even the ones that have not been posted as News)?

**Comment by wobba** [comment]
`PromotedState:2` is what you are looking for.
